### ISSUE TITLE
Textual differences in Romans

### ISSUE THREAD

**Issue by DavidHaslam** [BeTheLight/ENG-B1-Anderson1864-pd-USFM#8]
**Romans 1:17** should read:
```
\v 17 For in it the righteousness of God by faith is revealed, in order to faith: as it is written, He that is justified by faith, shall live.
```
The phrase "God’s plan of justifying men" is not found in the **1866** printing.

There are 4 more locations in Romans where the above phrase or similar needs to be replaced.

- Romans 3:5
- Romans 3:21
- Romans 3:22
- Romans 10:3 (2x)

See also [MOD-348](https://tracker.crosswire.org/projects/MOD/issues/MOD-348) on the CrossWire tracker.

**Comment by DavidHaslam** [comment]
I am already in the process of making corrections to the **Editing** branch in my fork of this repository.

_A pull request will be issued in due time_.

I have uploaded a PDF of a scanned facsimile for the 1866 printing to my forked repository.
Meanwhile, here's a [screenshot](https://www.dropbox.com/s/csg6qhpyji8g287/Screenshot%202019-02-02%2015.46.27.png?dl=0) of the page containing **Romans 1:17**.

NB. I rather suspect that the peculiar wording is from a different edition, possibly even an early 20th century revision made by the inheritors of H T Anderson's literary estate.

**Comment by DavidHaslam** [comment]
Merged pull request #9 

This textual change was the only one that could not be classified as correcting earlier OCR / spellcheck errors.
